### ISSUE TITLE
event not defined

### ISSUE THREAD

**Issue by innerop** [kolson25/WebRTC-Multi-Peer-Video-Audio#4]
https://github.com/kolson25/WebRTC-Multi-Peer-Video-Audio/blob/2ab22374f5e65ccbe5f0e2133d10521918992217/client/webrtc.js#L50

I haven't run this but I don't see how it would run without throwing an error

'event' is not defined anywhere

**Comment by IamTC** [comment]
You can accept the event in the function arguments

**Comment by innerop** [comment]
Yes obviously but it's not in the code, so I raised an issue to alert you that the code as it is could not work
